sqlglot/dialects/presto.py

```diff
@@ -546,3 +545,14 @@ def create_sql(self, expression: exp.Create) -> str:
             if kind == "VIEW" and schema.expressions:
                 expression.this.set("expressions", None)
             return super().create_sql(expression)
+
```

Comment:
is this duplicated anywhere?

Reply:
The `presto.py::strposition_sql` mirrors the `dialect.py::str_position_sql`. I didn't want to include `instance` generation in the base Dialect as this would mess up transpilations from Presto/Trino to other dialects, but we could merge these 2 by using a new (optional, defaults to False) boolean argument to choose when to generate the `instance` arg

Reply:
why don’t you just call the dialect version? why did you reimplement it?

Reply:
The Dialect version does not support generation of `instance` as it's currently implemented, so any Presto/Trino roundtrips or transpilations between each other would incorrectly drop it. However, as  the Dialect's `str_to_position_sql` generates exactly the same functions as Presto/Trino would, in my upcoming commit I've merged these 2 functions, hopefully that should be better.